pkg/cmd/server/kubernetes/node/node_config.go

```diff
@@ -325,24 +330,17 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 			dnsConfig.Nameservers = options.DNSNameservers
 		}
 
-		services, serviceStore := dns.NewCachedServiceAccessorAndStore()
-		endpoints, endpointsStore := dns.NewCachedEndpointsAccessorAndStore()
-		if !enableProxy {
```

Comment:
Previously, if we had proxy enabled, we would create our own copy of a reflector/cache on endpoints that we would share between dns and proxy. We're now using the kube shared informers to do so.